packages/terra-graphs/src/components/Graph.module.scss

```diff
@@ -0,0 +1,401 @@
+@import 'Variables.module';
+@import 'colors.module';
+
+svg:not(:root) {
```

Comment:
This global svg style also looks suspicious, but I'm not sure what impact this would have.

Reply:
Added [here](https://github.com/cerner/terra-graphs/pull/106/commits/8f1b7d01eb1f006f648f922b7b0ca25095b52fb2#diff-1bc65e5f27445458535bd92715dbb3b3ac661a88b2d165630603985a76ae4091R11).